solr/solr-ref-guide/modules/deployment-guide/pages/jwt-authentication-plugin.adoc

```diff
@@ -232,14 +239,21 @@ curl http://localhost:8983/solr/admin/authentication -H 'Content-type:applicatio
 
 [source,bash]
 ----
-curl http://localhost:8983/api/cluster/security/authentication -H 'Content-type:application/json' -H 'Authorization: Bearer xxx.yyy.zzz' -d -d '{"set-property": {"blockUnknown":true, "wellKnownUrl": "https://example.com/.well-knwon/openid-configuration", "scope": "solr:read solr:write"}}'
```

Comment:
`s/-d -d/-d` is the typo being fixed here. `git grep "\-d \-d"` finds no other cases of it.

Reply:
great...   Does that mean you are good with this change?   I meant to commit this twice, once on fixing the `-d -d` issue, and then seperately the reformatting changes.

Reply:
yes, good to go. thanks for spotting and fixing this!